adminStuff/addNewPost.php

```diff
@@ -0,0 +1,35 @@
+<?php
+include('config/init.php');
+if(isset($_REQUEST['submitItem'])){
+    $errors = array();
+
+    if(!@$_REQUEST['blogTitle']){
+        $errors['blogTitle'] = "*Required*";
+    }
+
+    if(!@$_REQUEST['body']){
+        $errors['body'] = "*Required*";
+    }
+    if(!@$_REQUEST['tagName']){
+        $errors['tagName'] = "*Required*";
+    }
+    if(sizeof($errors) == 0){
+        echo addPostQuery($_REQUEST['blogTitle'], $_REQUEST['body'], $_REQUEST['tagName']);
+        exit;
+    }
+}
+
+echo echoAdminHeaderHtml("Admin Section", "Edit Post").
+        "</div><h2> New Post </h2><div class = 'forms'></br>
+            <form action='' method='post'>
+            <input type='hidden' name='dateOfPost' id='dateOfPost' value=".getPostDateTime(). " />".
+            singleLineTextBox('Blog Post Title', 'blogTitle','Blog Title', @$_REQUEST['blogTitle'], @$errors['blogTitle'])."
+            ".textareaTextBox ('Blog Post', 'body', 'Type blog post here...', @$_REQUEST['body'], @$errors['body'])."
+            ".dropDownMenu ('Tag(s)', 'tagName', 'Select Tag', tagDropDownMenu(), @$errors['tagName'])."
+            <input type='hidden' name='articleid' id='articleid' />".
```

Comment:
unless I'm missing something, you aren't passing a value with this hidden input, so I don't think it's doing anything